raft/node_test.go

```diff
@@ -321,7 +321,15 @@ func TestNodeStart(t *testing.T) {
 		},
 	}
 	storage := NewMemoryStorage()
-	n := StartNode(1, []Peer{{ID: 1}}, 10, 1, storage)
+	c := &Config{
+		ID:              1,
+		ElectionTick:    10,
+		HeartbeatTick:   1,
+		Storage:         storage,
+		MaxSizePerMsg:   noLimit,
+		MaxInflightMsgs: 256,
```

Comment:
this is in raft pkg, not etcd pkg. also we set the const in etcdserver with a reason. they are not default values.